docs/developer/guidelines/dashboards.md

```diff
@@ -54,128 +49,115 @@ Ensure logos render correctly on the Dashboard List page and within the preset d
 1. Research the metrics supported by the integration and consider grouping them in relevant categories.
 Important metrics that are key to the performance and overview of the integration should be at the top.
 
-1. Ensure good design by keeping labels and widgets aligned with consistent spacing and reasonable size.
+### Rules of Thumb
```

Comment:
```suggestion
```

Reply:
Deleting because rules of thumb are "general" best practices (which already has a heading). If you need to distinguish the points below from the items in "General," the heading should be more specific about how they differ. (Also avoiding using the very colloquial English phrase.)